programs/bpf_loader/src/syscalls.rs

```diff
@@ -1,5 +1,8 @@
 use crate::{alloc, BpfError};
 use alloc::Alloc;
+use blake3::traits::digest::Digest;
```

Comment:
Are you using `cargo fmt` on these changes?

Reply:
Fixed in 03af7519e1ca7a29ed2ee25c2cd6b31e0d4e99fd